### PR TITLE
165447564 Chore update API doc

### PR DESCRIPTION
**Pull request by Joyce-O**
#### Description
This PR implements updating API documentation

#### Type of Change
Chore
- update app endpoints information
#### Pivotal Tracker
#165447564

### CODE REVIEW

**Comment by innocentEdosa** [comment]
LGTM

File: documentation.json [hunk 1]
```diff
@@ -1,107 +1,2001 @@
...
+           "example": 200
+         },
+         "message": {
+           "type": "string",
+           "example": "successful"
+         }
+       }
+     }
+   }
+ }
```
Comment (kayroy247) [inline documentation.json:2001]: Good job with the documentation. You may need to add a new line here.

**Comment by Bobjayafam** [comment]
LGTM

**Comment by kossy360** [comment]
LGTM

**Pull request merged**